src/VscPreviewExtension/.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+out/
+src/*
+test/
+media/*
+DfmParse/*
```

Comment:
how about we add the content later? seems none of the folder is added in this review?